src/data/list/palindrome.lean

```diff
@@ -0,0 +1,64 @@
+/-
+Copyright (c) 2020 Google LLC. All rights reserved.
```

Comment:
What is this Google LLC?!

Reply:
This is still in my personal time, not for work :sweat_smile: But company policy says that code belongs to them by default, and I can't be bothered arguing with the lawyers to get an exemption.

Reply:
Does @lambda-fairy work at Google?

> Although Google generally owns the IP rights to any development done during your employment, the open source license provides rights and responsibilities that protect the future use of the code for both you and Google.

https://opensource.google/docs/creating/

Reply:
I’m pretty sure this should not be marked as resolved. Surely the license issue is a Big Deal.

Reply:
If the code belongs to them, then you need their approval to release it under the Apache 2.0 license.

Reply:
You're right to be concerned, but I've already consulted the legal team about this – both about the copyright header / license, and potential conflicts of interest (as Lean 3 used to be a Microsoft project). They're aware of and are okay with my approach.

> If the code belongs to them, then you need their approval to release it under the Apache 2.0 license.

At Google, contributions to projects with compatible licenses do not need prior approval, unless there is a potential conflict of interest (which I have confirmation that this is not).

(Sorry about the confusion before; by "get an exemption" I meant getting a copyright release, which is unnecessary in this case.)

Reply:
Seems reasonable to me.